### ISSUE TITLE
DAO NACK not sending if node not present in nbr list

### ISSUE THREAD

**Issue by bbhagasra1** [contiki-os/contiki#1762]
for tcpoutput  if NBR not present  no packet process
nbr = uip_ds6_nbr_lookup(nexthop);
    if(nbr == NULL) {
# if UIP_ND6_SEND_NA

```
  if((nbr = uip_ds6_nbr_add(nexthop, NULL, 0, NBR_INCOMPLETE, NBR_TABLE_REASON_IPV6_ND, NULL)) == NULL) {
    uip_clear_buf();
    return;
  } else {
```
# if UIP_CONF

**Comment by joakimeriksson** [comment]
Correct. That is in a sense intentionally but not perfect. It is due to the RFC requiring that there should be a reachable neighbor in the cache before sending any packets to it. I would like to have a way to get DAO NACK sent anyways since we know that the neighbor is reachable because it is sending us a DAO - which means - it heard us - we heard it => reachable. I have a plan för a "temporary" neighbor entry that even with a strict neighbor policy that do not allow removal of any of the already existing neighbours and therefore not able to add this neighbor - we should be able to send the DAO ACK by installing the destination as a reachable neighbor in this "temporary" entry.
